client/packages/common/src/intl/locales/en/common.json

```diff
@@ -43,6 +43,7 @@
   "error.no-results": "Nothing here",
   "error.something-wrong": "Oops! Something's gone wrong.",
   "error.unable-to-load-data": "An error has occurred: there was a problem loading the data.",
+  "error.problem-saving": "There was a problem saving the data",
```

Comment:
:nit the message is inconsistent in format to the one above. Either could change: for me, I'd change this one to `An error occurred: there was a problem saving the data.`  the preamble softens it a bit, and there's a fullstop on the other one

Reply:
Changed it to:

Header: Missing data (re-using existing key)
Text: Can't access detailed information for this patient